migrations/20201015005801-update-user-roles.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
```

Comment:
@petorsong and I realized that we cannot use `Promise.all` for this particular change. When we put all these operations in one promise, it is not guaranteed that these queries will execute in the order that you have specified here. For example, it is possible that the `accessLevel` column is deleted before any of your update queries are executed. Since some of these queries are inter-related, you will need to use promise chaining as specified in the code changes in #62.

Reply:
yep I will fix this. I set up promise chaining but it got really messy and I decided to be lazy and try `Promise.all` anyway. it somehow executes properly but def not a good way to do it